webrender/src/device/gfx/device.rs

```diff
@@ -479,21 +479,19 @@ impl<B: hal::Backend> Device<B> {
         let pipeline_requirements: FastHashMap<String, PipelineRequirements> =
             from_str(&shader_source::PIPELINES).expect("Failed to load pipeline requirements");
 
-        let mut descriptor_pools_per_draw = SmallVec::new();
+        let descriptor_pools_per_draw = DescriptorPools::new(
+            &device,
+            descriptor_count.unwrap_or(DESCRIPTOR_COUNT),
```

Comment:
we could remove this if we start using rendy-descriptor. Wgpu-native has been ported to Rendy recently - we hit a few bumps, but otherwise things are working well.